apps/ui/src/components/Modal/DelegationConfig.vue

```diff
@@ -52,12 +53,12 @@ const definition = computed(() => {
       },
       apiType: {
         type: ['string', 'null'],
-        enum: [null, 'governor-subgraph'],
+        enum: [null, ...props.availableDelegationTypes],
```

Comment:
Not to sure about those changes, you won't reach this modal for offchain spaces, but if you could you would have mismatch between enum and options (we don't have option for `delegate-registry` so we made changes that are only half-compatible).

Maybe it would be better to not touch this entirely.